src/styles/blocks/stars.css

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url('../../images/star.svg');
+  background-repeat: no-repeat;
+  background-position: 50%;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). If you have multiple `.stars__star` elements stacked vertically, this rule may cause issues with margin collapsing. Consider using margin-top or margin-bottom consistently for spacing between elements.